<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_homepage</class>
 <widget class="QWidget" name="admin_homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>710</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>1</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #0C120C;
}</string>
   </property>
   <widget class="QLabel" name="labelFirstName_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: white;
}</string>
    </property>
    <property name="text">
     <string>Bienvenue Admin,</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_20" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #181F1C;
border-radius: 10px; 
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="ActionBtn_12" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>260</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <widget class="QToolButton" name="logcl_adm">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>260</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #181F1C;
	border-radius: 5px; 
}

QToolButton:hover{
background:#595554;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ActionBtnName_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>7</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Small</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: white;
background: transparent;
}</string>
     </property>
     <property name="text">
      <string>Consulter l'historique</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_28">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>38</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
border-radius:45px;
background:transparent;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/assets/people2.png</normaloff>:/assets/people2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_25" native="true">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>30</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #181F1C;
border-radius: 10px; 
}</string>
    </property>
    <widget class="QToolButton" name="adminChange">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
border-radius:45px;
}

QToolButton:hover{
border-radius:45px;
background:red;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/assets/logout.png</normaloff>:/assets/logout.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ActionBtn_11" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>260</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <widget class="QToolButton" name="crprofil_adm">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>260</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #181F1C;
	border-radius: 5px; 
}

QToolButton:hover{
background:#595554;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ActionBtnName_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>7</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Small</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: white;
background: transparent;
}</string>
     </property>
     <property name="text">
      <string>Créer / Associer un profil</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_27">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>38</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
border-radius:45px;
background:transparent;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/assets/people2.png</normaloff>:/assets/people2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelFirstName_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Variable Small</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: white;
}</string>
    </property>
    <property name="text">
     <string>Que souhaitez-vous faire aujourd'hui ?</string>
    </property>
   </widget>
   <widget class="QWidget" name="ActionBtn_9" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>260</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <widget class="QToolButton" name="cruser_adm">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>260</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #181F1C;
	border-radius: 5px; 
}

QToolButton:hover{
background:#595554;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ActionBtnName_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>7</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Small</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: white;
background: transparent;
}</string>
     </property>
     <property name="text">
      <string>Créer un utilisateur</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_23">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>38</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
border-radius:45px;
background:transparent;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/assets/people2.png</normaloff>:/assets/people2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ActionBtn_13" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>260</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <widget class="QToolButton" name="associer_adm">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>260</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #181F1C;
	border-radius: 5px; 
}

QToolButton:hover{
background:#595554;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ActionBtnName_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>7</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Small</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: white;
background: transparent;
}</string>
     </property>
     <property name="text">
      <string>Créer / Associer un compte</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_32">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>38</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
border-radius:45px;
background:transparent;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/assets/people2.png</normaloff>:/assets/people2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="LittleActionBtn" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <widget class="QToolButton" name="logoff">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>161</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #181F1C;
	border-radius: 5px; 
}

QToolButton:hover{
background:#595554;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="LittleActionBtnName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-2</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Small</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: white;
background: transparent;
}</string>
     </property>
     <property name="text">
      <string>Déconnexion</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="LittleActionIcon">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>4</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
border-radius:45px;
background:transparent;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/assets/logout2.png</normaloff>:/assets/logout2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="LittleActionBtn_2" native="true">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <widget class="QToolButton" name="switchRole">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>191</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #181F1C;
	border-radius: 5px; 
}

QToolButton:hover{
background:#595554;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="LittleActionBtnName_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-2</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Small</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: white;
background: transparent;
}</string>
     </property>
     <property name="text">
      <string>Mode Utilisateur</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="LittleActionIcon_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>4</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
border-radius:45px;
background:transparent;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressource.qrc">
       <normaloff>:/assets/unlock2.png</normaloff>:/assets/unlock2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>128</width>
       <height>128</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="topbar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(24,31, 28);
color:rgb(109, 114, 117);
border-top-right-radius: 10px;
border-top-left-radius: 10px;
</string>
   </property>
   <widget class="QLabel" name="label_64">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nohemi</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>BankRoot</string>
    </property>
   </widget>
   <widget class="QToolButton" name="reduced_adm">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
border-radius:45px;
}

QToolButton:hover{
border-radius:45px;
background:red;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/assets/reduce.png</normaloff>:/assets/reduce.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="fullscreen_adm">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
border-radius:45px;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/assets/fullscreen.png</normaloff>:/assets/fullscreen.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="close_adm">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{
border-radius:45px;
}

QToolButton:hover{
border-radius:45px;
background:red;
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/assets/leave.png</normaloff>:/assets/leave.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
